app/utils/get_filtered_records.py

```diff
@@ -159,23 +160,38 @@ def should_include(d, k, v):
     else:
         result = query_dicts
 
-    def date_filter(record, start_date=None, end_date=None):
+    def date_filter(record, start_date=None, end_date=None, sampling_date_type=True):
         status = True
 
         if start_date is not None:
-            status = status and record["sampling_end_date"] and start_date <= record["sampling_end_date"]
+            # If sampling_date_type is True, compare to the sampling start date
+            if sampling_date_type:
+                status = status and record["sampling_start_date"] and record["sampling_start_date"] >= start_date
```

Comment:
also this should be comparing against `sampling_end_date`, not `sampling_start_date`

Reply:
are you sure? we want to include the record if it's sampling start date is greater than the minimum start_date that we want, no? e.g. if start_date is Nov 10 2019, and the record sampling_start_date is Dec 10, 2019, we want to check if Dec 10, 2019 > Nov 10 2019

Reply:
As addressed on slack, my comment holds